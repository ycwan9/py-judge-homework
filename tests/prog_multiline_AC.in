rAySzCZzi7xLv5FSf1umWMVeElqdk5nijLcyYnKDir8m2W6JcxAuyRVwr1F2RhtwenHM1AV2t6Hv
7n5LYA+2zoYsG/ydlZlhIZEOMrzwrhc1xytB9QDCVXoxpFeWIR8+lrwJqxVzHaiYIvEgTHJ8qZsn
7N1x8hHAas0tbSxdYU3qfwX8fPUpjUGSR/i7Huca2z+GytTiOAjcPpJLCSGMbcrPJLfNfYu61byU
bwFwLB9zAnA0Wcvxkt1kAFJF9PrHP3FmkawD2xz/0bPOwWVrm3AIhCjDC0kBlcPOCQiuW1wAz5Lp
2+hkJihpzsPymBp28BWJZELdtYX0XlivnXPGHymXoz68m+udGQac9iqql8WVVElwpG0ddNSlt4ze
Vmz6HaoyN2H9uMqT+Gao1Lh/HBUSYgAWUW0YZ2jXbBRDJ+RvSoWRBWIyo+P7HiSE9+Zy0eKrJl7u
1GX+WDZvJXo9vGRw4/3kKs9HxEXheGlnuGSfm6/nRYMlFSBAxtibwefFpdno+uHBV9SyE+dJbhhT
U0ibgvfBrxi/WxRQQNpMsSP8m0VTLkHmSrBiSTACnP/OcBdUbLfUwmttJjYsAz4HWO0qataFnWEt
NjVk7GbJiiAM6mnGCmIoI7sqoswpmIycpdS00A3iHafVs8zuj/Ah2o1ZRWfHDXqjNm/s9i+ILe/X
MVlzsaKucRYqfg+7kEvQerxYdmnDS7I9g12rwlMz5lhzKiSBFcTyXWkFmtvsuUOafuR7cYTKGvOf
jDAWxhVDzyOaZg21XDnPzwiwFDEMDG8u7QWZQVmXK3irp8n0oKRrnh3CWAmiBWRPmdjDOVFFScm0
Aoga8v2VjqCU9uNyyi5DHfbV/fBFXcl8yoAinDV/xiIwm+Gsyo8KMgcAGXoCd+RtxZ38Jni3knPz
eKJBoUbHOLXvxCLg3w1TWm4rZx9XcjYJDooJMxkjf6+E99WPxoz7IMnP1zlz/It2w9qWYi9RfKl3
itPapDkCcpiixMAJJx3F2ryBp4B9xyr7SgphU6BDN7fnUzsR8xA/5He1BJIA9qUAPggtfs44jPwr
c36S8Pmt1BWzRpOSprfHiYatV77hEV3dyK+0mj3v5FrKDRBN5UKqxkZz+z7Y0g2yBAxml334sTsX
u5PgyRsLVvXOypsm6IswpRRIiMPo+3iyoBiDt9LTOvQUeSovv+nggiSXhU5LBVLLgIhs7QYJfmYs
SeH7Hs4ITXhNl/d/9GbbmYJcXJjLNZK1MO8qmsv40iwLDTv8bgLOjX3b5G9nZo0hKmV1npsIEwZF
NY+otRfZl2Wc9nme4jwd0wINjc6PrnZtbbskRp0up43n6RItwUuX8L7HaorGU27pPk8PY8wnW3YS
539+c83azI5IGfyPX7yaR2Nf+C+FrL/1pgaBiBI2MFTC3Ew7PyRmPyQPqs7RlJ5/XL164+WLfnBq
FJzTzmJSUGHoMwjIz8jejOSKXrsrPrB/vgF3H3gr2MFuLHgFFarEdmFpet+shhrbMIivi+VVlWxl
