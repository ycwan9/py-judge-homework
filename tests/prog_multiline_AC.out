prog_multiline_AC.in